graphql/resolvers.go

```diff
@@ -2283,7 +2283,7 @@ func (r *mutationResolver) SetTaskPriority(ctx context.Context, taskID string, p
 
 func (r *mutationResolver) OverrideTaskDependencies(ctx context.Context, taskID string) (*restModel.APITask, error) {
 	currentUser := MustHaveUser(ctx)
-	t, err := task.FindOneIdAndExecutionWithDisplayStatus(taskID, nil)
+	t, err := task.FindByIdExecution(taskID, nil)
```

Comment:
Actually sorry last thought -- I think we do need to be passing in execution here. A user may want to override dependencies on an old execution of a task; this function will always only return the most recent

Reply:
We actually don't allow any task actions if the task is not on the latest execution. Including overriding dependencies, I've added the same safety check to my pr. 
https://evergreen-staging.corp.mongodb.com/task/evg_ubuntu1604_test_cloud_patch_ccbdfbe1ca4c5e72baed4d0505b6b058566c89b4_61083e9a97b1d31e62c81c7b_21_08_02_18_51_19/0 
https://github.com/evergreen-ci/evergreen/blob/921e132cd5094bbae58151c550ac915110ca2c83/service/templates/task.html#L118

Reply:
Thanks for finding!